src/components/cylc/analysis/TimeSeries.vue

```diff
@@ -0,0 +1,442 @@
+<!--
+Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+  <Teleport
+    v-if="sortInputTeleportTarget"
+    :to="sortInputTeleportTarget"
+  >
```

Comment:
Could do with renaming to e.g. `toolbarTeleportTarget` to avoid confusion (I think this was copied from the box & whiskers plot)

Reply:
Done